guides/common/modules/proc_configuring-pull-transport.adoc

```diff
@@ -0,0 +1,31 @@
+[id="configuring-pull-provider_{context}"]
+= Configuring Remote Execution for Pull Client
+
+By default, Remote Execution uses SSH as the transport mechanism for the Script provider.
+However, Remote Execution also offers a pull-based transport option, which communicates with hosts via MQTT.
+This is a good option if you are migrating from `katello-agent` which also uses pull-based communication.
+
+If you wish to use Remote Execution with the pull-based transport, run the following `{foreman-installer}` command on each {SmartProxy}:
+
+[options="nowrap" subs="quotes,attributes"]
+----
+
+ # {installer-scenario-smartproxy} \
+  --foreman-proxy-plugin-remote-execution-script-mode pull-mqtt
+
+----
+
+The mode is configured per {SmartProxy}.
+Some {SmartProxies} may be configured to use `pull-mqtt` mode while others use `ssh`.
+If this is the case, it is possible that one remote job on a given host will use the pull client, and the next job on the same host will use SSH.
+If you wish to avoid this scenario, configure all {SmartProxies} to use the same mode.
+
+In `pull-mqtt` mode, hosts subscribe to the {SmartProxy} they are registered through for job notifications.
+Therefore, it is recommended to ensure that {ProjectServer} sends remote execution jobs to that same {SmartProxy}.
+To do this, in the {ProjectWebUI}, navigate to *Settings* > *Content*, and set the value of "Prefer registered through {SmartProxy} for remote execution" to *Yes*.
+
+After the pull-based transport is set up on each {SmartProxy}, it must also be configured on each host.
```

Comment:
"set up on each (proxy)" -> "set up on a (proxy)"  (as we can have ssh on another proxies, or alter one proxy from ssh to mqtt).

"on each host" -> "on each relevant host" ? To distinguish there might be hosts that will still use ssh mode?

Reply:
> "set up on each (proxy)" -> "set up on a (proxy)" (as we can have ssh on another proxies, or alter one proxy from ssh to mqtt).

:+1: 

> "on each host" -> "on each relevant host" ? To distinguish there might be hosts that will still use ssh mode?

If a given (proxy) uses pull-mqtt, doesn't that mean that all of its hosts must be configured with the pull client? So I'd propose leaving "each host" as-is.